File: codeforces/83.magical_sticks_[1371A]/magical_sticks.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int t;
  unsigned int n, max_num_sticks;
  cin >> t;
  while (t--) {
    cin >> n;
    if ((n % 2) == 0) {
      max_num_sticks = n / 2;
    }
    else {
      max_num_sticks = (n + 1) / 2;
    }
    cout << max_num_sticks << endl;
  }
  return 0;
}

File: codeforces/low_difficulty/difficulty_800/83.magical_sticks_[1371A]/magical_sticks.py
# Author: José Rodolfo (jric2002)
t = int(input())
while (t):
  n = int(input())
  max_num_sticks = 0
  if ((n % 2) == 0):
    max_num_sticks = int(n / 2)
  else:
    max_num_sticks = int((n + 1) / 2)
  print(max_num_sticks)
  t -=1